src/tools/pontoon/adding_new_project.md

```diff
@@ -21,7 +21,7 @@ Access Pontoon’s [admin console](https://mozilla-pontoon-staging.herokuapp.com
   * You can uncheck the `Locales can opt-in` checkbox to prevent localizers from requesting this specific project.
 * Repositories: select the type of repository and URL. Make sure to use SSH to allow write access. For example, if the repository is `https://github.com/meandavejustice/min-vid`, the URL should be `git@github.com:meandavejustice/min-vid.git`. You can use the *Clone or download* button in the repository page on GitHub, making sure that *Clone with SSH* is selected.
 * Leave the `Branch` field empty, unless developers asked to commit translations in a branch instead of *master*.
-* Download prefix: a URL prefix for downloading localized files. For GitHub repositories, select any localized file on GitHub, click `Raw` and replace locale code and the following bits in the URL with `{locale_code}`. For example, if the link is `https://raw.githubusercontent.com/bwinton/TabCenter-l10n/master/locales/en-US/addon.properties`, the field should be set to `https://github.com/bwinton/TabCenter-l10n/blob/master/locales/{locale_code}`.
+* Download prefix: a URL prefix for downloading localized files. For GitHub repositories, select any localized file on GitHub, click `Raw` and replace locale code and the following bits in the URL with `{locale_code}`. For example, if the link is `https://raw.githubusercontent.com/bwinton/TabCenter-l10n/master/locales/en-US/addon.properties`, the field should be set to `https://github.com/bwinton/TabCenter-l10n/blob/master/locales/{locale_code}`. If you use one, you need to select the project configuration file instead of a localized file.
```

Comment:
My first reaction was "if you use one" of what? Also, you don't really select the file here. What about this?

> If you use a project configuration file, you need to provide the path to the raw TOML file on GitHub, e.g. `https://raw.githubusercontent.com/mozilla/common-voice/main/l10n.toml`.

Also, should we change the name of that field in Pontoon? It's one thing to update the docs, but I think the field's name is currently very misleading.